online_store/gt_sport_v1.html

```diff
@@ -0,0 +1,120 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="column" id="h2title">goodTeens - Sports</h2> <p>
+<h2 class="column" id="h2title2">Online Store</h2>
+<br><br>
+<p class="a"></p>
+<section class="card" id="product1"> 
+</section>
+<p class="b"></p>
+<section class="card" id="product2"> 
+</section>
+<p class="b"></p>
+<section class="card" id="product3"> 
+</section>
+<p class="b"></p>
+<section class="card" id="product4"> 
+</section>
+</section>
+<p class="b"></p>
+<section class="card" id="product5"> 
+</section>
+
+<script>
+const sportProducts = [
+  {
+    imgSrc: "skateboard.jpg",
```

Comment:
where are these images? Did you add commit them?
<img width="351" alt="Screenshot 2020-04-30 at 13 08 58" src="https://user-images.githubusercontent.com/13404719/80703847-d3922100-8ae3-11ea-9c31-f0b213c38afe.png">

Reply:
Re-uploaded them. Probably deleted them by mistake when 'tidying up'.